src/scheduler/interface/scheduler.interface.ts

```diff
@@ -0,0 +1,27 @@
+import { ApiProperty, PartialType } from '@nestjs/swagger';
+
+//predefined recursion
+export enum RecurringInterval {
+  EVERY_DAY = 1,
```

Comment:
Why have to assign EVERY_DAY = 1 here

Reply:
i was concerned about having enum value = 0 since it is a falsy value.